src/app/core/messages/message.controller.js

```diff
@@ -132,27 +132,27 @@ exports.update = function (req, res) {
 
 // Delete
 exports.delete = function (req, res) {
-	const message = req.message;
-	Message.deleteOne({ _id: message._id })
-		.exec()
-		.catch((err) => {
-			util.catchError(res, err, () => {
-				res.status(200).json(message);
-			});
-		});
+	try {
+		const message = req.message;
+		Message.deleteOne({ _id: message._id }).exec();
```

Comment:
Need to add `await` to this line to wait for the completion of `Message.deleteOne`.  Otherwise no errors would be thrown/caught.